tests/integration/test_lock.py

```diff
@@ -791,3 +791,39 @@ def test_pipenv_respects_package_index_restrictions(PipenvInstance):
                                       'sha256:ec22d826a36ed72a7358ff3fe56cbd4ba69dd7a6718ffd450ff0e9df7a47ce6a'],
                            'index': 'local', 'version': '==2.19.1'}
         assert p.lockfile['default']['requests'] == expected_result
+
+
+@flaky
```

Comment:
We shouldn't be adding any new flaky tests.

Reply:
I will remove it. Frankly, I don't know what it does, I use it because I saw other tests functions use it as well.

Reply:
I think it was considered for tests that would sometimes fail to have them re-run, but we expect our tests to pass so its just an odd annotation.